<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportKey</class>
 <widget class="QFrame" name="ExportKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>116</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="TextLabel2">
     <property name="text">
      <string>When exporting the private key it should be encrypted.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exportPrivate">
     <property name="toolTip">
      <string>When exporting the private part, it should be encrypted.</string>
     </property>
     <property name="text">
      <string>E&amp;xport the private part of the Key too</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exportPkcs8">
     <property name="text">
      <string>Export as PKCS#8</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="encryptKey">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Encrypt the Key with a password</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
